src/getHumanAge.test.js

```diff
@@ -7,4 +7,65 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test ('the result should return an array', () => {
+    const result = getHumanAge();
+    expect(result)
+    .toBeInstanceOf(Array);
+  });
+
+  test ('should return an array with two elements', () => {
+    const result = getHumanAge();
```

Comment:
You are not passing any arguments to the function getHumanAge. Please provide arguments to the function.